ha/cli/cortxha.py

```diff
@@ -55,12 +55,12 @@ def __init__(self):
         Initialization of HA CLI.
         """
         # TODO Check product env and load specific conf
-        Conf.init()
-        Conf.load(const.RESOURCE_GLOBAL_INDEX, Json(const.RESOURCE_SCHEMA))
-        Conf.load(const.HA_GLOBAL_INDEX, Yaml(const.HA_CONFIG_FILE))
-        log_path = Conf.get(const.HA_GLOBAL_INDEX, "LOG.path")
-        log_level = Conf.get(const.HA_GLOBAL_INDEX, "LOG.level")
-        Log.init(service_name='cortxha', log_path=log_path, level=log_level)
+        ConfigManager.init('cortxha')
+
+        # get version from ha.conf
+        version = ConfigManager.get_major_version()
```

Comment:
Major version is just the first field - 2. The complete string is version. What you are returning from the ConfigManager is version, no the major version.
Your code in here, should be as following -
`self._version = ConfigManager.get_major_version()`

Everything else should go to ConfigManager.

Reply:
done